modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -86,6 +131,27 @@ export default class ParticipantConnectionStatus {
          */
         this.rtcMutedTimestamp = { };
         logger.info(`RtcMuteTimeout set to: ${this.rtcMuteTimeout}`);
+
+        /**
+         * This map holds the timestamps indicating when participant's video
+         * entered lastN set. Participants entering lastN will have connection
+         * status restoring and when we start receiving video will become
+         * active, but if video is not received for certain time
+         * {@link DEFAULT_RESTORING_TIMEOUT} that participant connection status
+         * will become interrupted.
+         *
+         * @type {Object.<string, number>}
+         */
+        this.enteredLastNTimestamp = {};
+
+        /**
+         * A map of the "endpoint ID"(which corresponds to the resource part
+         * of MUC JID(nickname)) to the restoring timeout callback IDs
+         * scheduled using window.setTimeout.
+         *
+         * @type {Object.<string, number>}
+         */
+        this.restoringTimers = [];
```

Comment:
It does work, because arrays are objects:

```
> var x = []
undefined
> x['foo'] = 'bar'
'bar'
> x
[ foo: 'bar' ]
> typeof x
'object'
> Array.isArray(x)
true
>
```